src/main/kotlin/net/maizegenetics/phgv2/cli/SetupEnvironment.kt

```diff
@@ -16,7 +16,7 @@ import java.util.stream.Collectors
  * Check if this environment already exists.  If an error occurs when
  * creating it, do we assume it exists?
  */
-class SetupEnvironment : CliktCommand() {
+class SetupEnvironment : CliktCommand(help="create conda environment for PHG") {
```

Comment:
Maybe this was discussed in another PR, but will this also exported into the `Phg().subcommands()` method?

Reply:
Yes looks like this was forgotten about.  I will add it in.